test/integration/e2eIntegrationTests.coffee

```diff
@@ -861,13 +862,14 @@ describe "e2e Integration Tests", ->
               if err
                 done err
               else
-                Transaction.findOne {}, (err, res) ->
-                  res.status.should.be.equal mediatorResponse.status
-                  res.orchestrations.length.should.be.exactly 1
-                  res.orchestrations[0].name.should.be.equal mediatorResponse.orchestrations[0].name
-                  should.exist res.properties
-                  res.properties.orderId.should.be.equal mediatorResponse.properties.orderId
-                  done()
+                Transaction.findOne {}, (err1, res1) ->
```

Comment:
I could not find a cleaner way to delay checking for the updated status, the test was checking for the status before it had been updated,.